### DISCUSSION TITLE
Phoscon “Kobold” Zigbee 3.0 smart dimmer switch module from Dresden Elektronik?

### DISCUSSION THREAD

**Discussion by Hedda** [Koenkk/zigbee2mqtt#14210] [Devices]
Anyone tried these Phoscon Kobold branded smart dimmer switch modules from Dresden Elektronik?

[https://phoscon.de/en/kobold](https://phoscon.de/en/kobold)

![image](https://user-images.githubusercontent.com/6320001/192543024-a6e9db5c-35ba-4722-8e54-dc79ee115dd2.png)

Has anyone tested these Phoscon Kobold modules in Europe with relatively low-power LED lights?

They look a lot like other unbranded Zigbee 3.0 smart dimmer switch modules from China but coming from Dresden Elektronik / Phoscon I expect high quality and firmware to be maintained for a long time.

[https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/OTA-Image-Types---Firmware-versions#dresden-elektronik](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/OTA-Image-Types---Firmware-versions#dresden-elektronik)

I have during the past 10 years tried what feels like loads of similar dimmer switch modules from many other manufacturers and all of them have had some problems smoothly dimming lower than around 15% with the different low-power dumb dimmable LED lamps that are not smart lightbulbs, (the best dimmers for I found so far for low-power dumb dimmable LED lamp loads is Qubino Flush Dimmer and Qubino Mini Dimmer which actually use Z-Wave).

|Link|Country|
| --- | --- |
|[Amazon.de](https://www.amazon.de/dp/B09K81HJ69)|Germany|
|[Amazon.nl](https://www.amazon.nl/dp/B09K81HJ69)|Netherlands|
|[Amazon.pl](https://www.amazon.pl/dp/B09K81HJ69)|Poland|
|[Amazon.se](https://www.amazon.se/dp/B09K81HJ69)|Sweden|
|[Amazon.fr](https://www.amazon.fr/dp/B09K81HJ69)|France|
|[Amazon.es](https://www.amazon.es/dp/B09K81HJ69)|Spain|
|[Amazon.it](https://www.amazon.it/dp/B09K81HJ69)|Italy|

![image](https://user-images.githubusercontent.com/6320001/192543119-a3ee78ff-dfca-4920-a48c-3c8d95346055.png)

![image](https://user-images.githubusercontent.com/6320001/192543078-4af7a9f0-ee68-45ed-a9e2-fcf53cd37c99.png)

**Comment by Hedda** [comment]
Here is by the way the related device description file (DDF) from the deCONZ integration:

https://github.com/dresden-elektronik/deconz-rest-plugin/blob/master/devices/dresden_elektronik/kobold.json

> They look a lot like other unbranded Zigbee 3.0 smart dimmer switch modules from China but coming from Dresden Elektronik / Phoscon I expect high quality and firmware to be maintained for a long time.

FYI, dresden elektronik (deCONZ/Phoscan) Zigbee developer [manup](https://github.com/manup) posted this related comment about this “Kobold” module hardware:

"**_Side note it looks similar to other products and is in fact based on the same hardware from Sunricher, but we've developed a own firmware for it. Main difference is that Kobold has a switch endpoint (EP: 2) so bindings for on/off and brightness control can be created for groups. With that the unit can control other lights also when the gateway is switched off._**"

So it does indeed sound like it uses same internal hardware as Sunricher SR-ZG9101SAC-HP AC phase-cut dimmer but different firmware:

https://www.sunricher.com/zigbee-ac-phase-cut-dimmer-sr-zg9101sac-hp.html

**Reply by cromefire** [reply]
Is there any way to get the actions working with that? I cant wind which attributes to read from their DDL file. I identified that they where using the cluster genOnOff (0x0006) for the endpoint 2, but I cant find the correct attribute to read.
